.github/workflows/ci.yaml

```diff
@@ -130,3 +130,34 @@ jobs:
         with:
           name: manager_linux
           path: convenient_test_manager.tar.gz
+
+  build_manager_headless_cli:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+          architecture: x64
+
+      - name: Install dependencies
+        run: dart pub get
+        working-directory: packages/convenient_test_manager_dart
+
+      - name: Dart Compile
+        run: |
+          dart compile exe ./bin/convenient_test_manager_dart.dart -o manager_headless
+          dart compile kernel ./bin/convenient_test_manager_dart.dart -o manager_headless_kernel
+        working-directory: packages/convenient_test_manager_dart
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: manager_linux-mac_headless_cli
```

Comment:
Maybe this is only useable on linux, not on mac. This is probably because `dart compile exe` does not support cross compilation, and mac has different executable format than linux.

So maybe we should rename as sth like manager_headless_linux

e.g. the file, if executed on a mac, gives:

```
/Users/.../Downloads/manager_headless          
zsh: exec format error: /Users/.../Downloads/manager_headless

Reply:
I don't have a mac to test but I was going based off this comment: https://github.com/dart-lang/sdk/issues/28617#issuecomment-401427781

That user claims, ubuntu and mac should be cross compatible. I added in the kernel file so there is at least something that is cross compatible to all platforms. :)

Reply:
I see. Unfortunately it does not work according to my experiment.